### PR TITLE
Add basic topic config for energy-budget-plan namespace

### PR DESCRIPTION
**Pull request by cdempsie**
- Creates 3 topics for energy-budget-plan: `energy-budget-plan.eqdb-loader`, `energy-budget-plan.budget-plan` and `energy-budget-plan.customer-change`
- Configures an EQDB loader process as producer on `energy-budget-plan.eqdb-loader`
- Configures a Budget Plan Fabricator process as the consumer of `energy-budget-plan.eqdb-loader` and producer on `energy-budget-plan.customer-change`
- Configures Budge Plan Calculator as consumer and producer of `energy-budget-plan.budget-plan`
- Cert config in wip PR https://github.com/utilitywarehouse/kubernetes-manifests/pull/85143

### CODE REVIEW

File: dev-aws/kafka-shared-msk/energy-budget-plan/energy-budget-plan.tf [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+  produce_topics   = [kafka_topic.eqdb_loader.name]
+  cert_common_name = "energy-budget-plan/eqdb-loader"
+}
+
+module "budget_plan_fabricator" {
+  source           = "../../../modules/tls-app"
+  consume_topics   = { (kafka_topic.eqdb_loader.name) : "energy-budget-plan.eqdb-fabricator-v1" }
+  produce_topics   = [kafka_topic.customer_change.name]
+  cert_common_name = "energy-budget-plan/eqdb-fabricator"
+}
```
Comment (oliverpauffley) [inline dev-aws/kafka-shared-msk/energy-budget-plan/energy-budget-plan.tf:64]: The way this is currently built it consumes its own messages. We can change that of course but maybe we add in consume on the `customer_change` too?

Comment (cdempsie) [reply dev-aws/kafka-shared-msk/energy-budget-plan/energy-budget-plan.tf:64]: Sure, can do that while I am here.

Comment (cdempsie) [reply dev-aws/kafka-shared-msk/energy-budget-plan/energy-budget-plan.tf:64]: @utilitywarehouse/pubsub do consumer group names have to be unique or can I specify the same consumer group name on different topics?

Comment (sbuliarca) [reply dev-aws/kafka-shared-msk/energy-budget-plan/energy-budget-plan.tf:64]: Hmmm, this is a good question. The way the tls-app is module is working right now, doing that would create the group_acl resource twice https://github.com/utilitywarehouse/kafka-cluster-config/blob/main/modules/tls-app/main.tf#L13-L21, so I think Terraform would be confused bu that.

I'll look into it and see if we can adjust that, but for now I would suggest a different consumer group.

**Review by sbuliarca — APPROVED** [review]
LGTM

File: dev-aws/kafka-shared-msk/energy-budget-plan/energy-budget-plan.tf (hunk 1 above)
Comment (cdempsie) [reply dev-aws/kafka-shared-msk/energy-budget-plan/energy-budget-plan.tf]: I've made the consumer groups unique for now.

**Pull request merged**